dev/traffic_router/conf/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+ -->
+<Configuration>
+	<Appenders>
+		<Console name="stdout" target="SYSTEM_OUT">
+			<PatternLayout pattern="%m%n"/>
+			<ThresholdFilter level="ALL"/>
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.apache.traffic_control.traffic_router" level="INFO" additivity="false">
+			<AppenderRef ref="stdout"/>
+		</Logger>
```

Comment:
The appender used for the `org.apache.traffic_control.traffic_router` logger should include the class name and optionally other info that the default `log4j2.xml` includes:
https://github.com/apache/trafficcontrol/blob/dfca6b12a736ee0430b27183acf2db911f85b65b/traffic_router/core/src/main/conf/log4j2.xml#L34